### ISSUE TITLE
[Question] Changelog block

### ISSUE THREAD

**Issue by BentCoder** [Redocly/redoc#1477]
Hi,

Is there any way of adding a Changelog **at the bottom** of the menu as shown below?

Thanks

<img width="280" alt="Screen Shot 2020-12-04 at 17 52 23" src="https://user-images.githubusercontent.com/8837278/101197347-e9f3f280-3659-11eb-9816-0a5300778a32.png">

**Comment by bu4ak** [comment]
Same question

**Comment by lornajane** [comment]
No, we don't have this functionality and I'm afraid it's not on our roadmap. Most of our users hold the changelog information elsewhere in their API portal, so it's not a priority for us to support it.
